helper/resource/testing_sets.go

```diff
@@ -113,6 +164,39 @@ func TestMatchTypeSetElemNestedAttrs(name, attr string, values map[string]*regex
 //
 // Use this function over SDK provided TestCheckFunctions when validating a
 // TypeSet where its elements are a simple value
+
+// TestCheckTypeSetElemAttr ensures a specific value is stored in state for the
```

Comment:
`for the for the`

Reply:
Will fix this one manually as trying to do 80 character line fixes in GitHub's suggestion box is a silly exercise.

Reply:
Fixed in https://github.com/hashicorp/terraform-plugin-sdk/pull/916/commits/1018c5a5dbc2882d22472fc6a81766747133920a